packages/beacon-node/src/execution/engine/interface.ts

```diff
@@ -30,7 +30,7 @@ export enum ExecutePayloadStatus {
   UNSAFE_OPTIMISTIC_STATUS = "UNSAFE_OPTIMISTIC_STATUS",
 }
 
-export enum ExecutionEngineState {
+export enum ExecutionState {
```

Comment:
i am thinking if `ExecutionState` might cause confusion over reference to actual execution state, in that case original `ExecutionEngineState` is preferable

Reply:
If I use `ExecutionEngineState`, then would prefer to use the same vocabulary/wording to other places for variables and functions. That will lead to almost similar to what we had earlier. Will it be ok?